mitiq/pec/pec.py

```diff
@@ -102,6 +103,13 @@ def execute_with_pec(
     if not isinstance(num_samples, int):
         num_samples = int((norm / precision) ** 2)
 
+    # Issue warning for very large sample size
+    if num_samples > 10 ** 5:
+        warn(
+            "Sample size in PEC is very large.It may take serveral mintues.",
```

Comment:
```suggestion
            "The number of PEC samples is very large. It may take several minutes."
            " It may be necessary to reduce 'precision' or 'num_samples'.",
```
We could add this suggestion for the user. 

I apologize if this change creates flake8 problems. If so, they should be easy to fix.